### ISSUE TITLE
Change in behavior between 1.6.4 and 1.6.5 for getErrorMessages

### ISSUE THREAD

**Issue by mjclemente** [nahsra/antisamy#151]
First - thank you for all the work that goes into Antisamy - really appreciate the active development and work that goes into it!

To begin - I did read the [README](https://github.com/nahsra/antisamy/#5-analyzing-cleanresults) and understand that `getErrorMessages()` does not answer the question as to whether input is safe or not. In light of that, you may not care about this change in behavior, but I thought it worth noting.

In version 1.6.4, the following input triggered an error message:

```html
<img
    style="FLOAT: right; MARGIN: 0px 0px 10px 10px; WIDTH: 150px; CURSOR: hand; HEIGHT: 100px" alt=""
    src="http://www.charityadvantage.com/ChildrensmuseumEaston/images/BookswithBill.jpg" border="0" />
```

Calling `getErrorMessages()` would return:

> The img tag had a style attribute, "CURSOR", that could not be allowed for security reasons.

As of 1.6.5, the message is no longer returned - though to be clear, the cursor property is still removed, as expected.

Is this change expected? 

Thanks again!

**Comment by davewichers** [comment]
@spassarop - Your thoughts on this?

**Comment by spassarop** [comment]
Hi @mjclemente. You're right, a constructor for `CssHandler` class was added and it was not using the error messages list it had as parameter. The constructor would always create a new empty `ArrayList`. 

The fix is easy so expect to find it fixed for 1.7.0, I'll make the change in a moment.

@davewichers, the change was made in [this commit](https://github.com/nahsra/antisamy/commit/d8fa2f430410f5b7e36b68968e25f3b9278371ff#diff-26fc9797189a5be44e00fe0a4f5db1e429c6b0c0e51915cb3cadf7701ad3615aR170). Do you remember why it was left with an empty list in the first place? I think you asked me to review the changes but I didn't note that.

**Comment by davewichers** [comment]
@spassarop - Ii don't recall why. Probably a mistake on my part. If you are going to edit copyright dates, you might as well say its 2022, not 2021 :-)

**Comment by spassarop** [comment]
Date edited on test file

**Comment by davewichers** [comment]
@mjclemente - Thanks for reporting this. This will be fixed in the 1.7.0 release.

**Comment by davewichers** [comment]
Fix included in v1.6.6 release.
